akd/src/directory.rs

```diff
@@ -510,7 +510,7 @@ mod tests {
         let mut akd = Directory::<_>::new::<Blake3>(&db).await?;
 
         akd.publish::<Blake3>(
-            vec![(AkdLabel("hello".to_string()), AkdValue("world".to_string()))],
+            vec![(AkdLabel(Vec::from("hello")), AkdValue(Vec::from("world")))],
```

Comment:
It's tests so I don't really care, but generally will this give us utf8? We're not specifying any kind of encoding, so it'll be whatever this impl has chosen.

Reply:
Yes: https://doc.rust-lang.org/std/vec/struct.Vec.html#impl-From%3C%26%27_%20str%3E

Internally, the `From` implementation is calling `as_bytes` on the `&str` and then loading it like any other array.